src/Controller/PageAdminController.php

```diff
@@ -294,7 +294,7 @@ public function composeContainerShowAction(Request $request): Response
             }
             $container = $template->getContainer($blockCode);
 
-            if (null !== $container) {
+            if (null !== $container && \count($container['blocks']) > 0) {
```

Comment:
I made a mistake here: https://github.com/sonata-project/SonataPageBundle/pull/1566/files#diff-35e8c6ccc40ddfb4183935e83027e779cce76d90ecdcf092fca7d4f32004d889R261

We check for count > 0 since if we have 0 elements, we do not remove anything from the default block list, otherwise you end up with an empty list of blocks.